### ISSUE TITLE
WIP: Make stereo display support optional at build time

### ISSUE THREAD

**Issue by fifteenhex** [directfb2/DirectFB2#136]
Issue for something I'm working on:

3D displays are out of fashion and I don't think many DFB2 users are going to be using them.
So allow the code for that to be left out at compile time.

**Comment by caramelli** [comment]
Yes, we can provide an option to disable support for 3D displays:

	option('stereo',
	       type: 'boolean',
	       description: 'Stereoscopic display support')

If anyone has any setup and information regarding stereoscopic display, I'd be interested in testing it.
